docs/software/webserver_stack.md

```diff
@@ -1,25 +1,137 @@
-# Webserver & Web Stacks
+# Web Development
 
-## Overview
+## Stacks for web development
 
-- [LAMP Webserver - Apache2 / MariaDB / PHP](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=10#p52)  
+- [**LAMP Webserver - Apache2 / MariaDB / PHP**](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=10#p52)  
```

Comment:
Change link from (https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=10#p52) to (#lamp-webserver-apache2-mariadb-php)